packages/office-ui-fabric-react/src/components/TextField/TextField.base.tsx

```diff
@@ -503,7 +518,7 @@ export class TextFieldBase extends React.Component<ITextFieldProps, ITextFieldSt
     }
 
     this._latestValidateValue = value;
-    const onGetErrorMessage = this.props.onGetErrorMessage as (value: string) => string | PromiseLike<string> | undefined;
```

Comment:
Why is this being removed?

Reply:
Because in this case there's no reason to cast, all that's needed is a non-null assertion (`!` operator)--but I guess whoever wrote this didn't know about that?